2021-preseason/src/main/java/frc/robot/Robot.java

```diff
@@ -35,8 +44,13 @@ public void robotInit() {
     m_chooser.setDefaultOption("Default Auto", kDefaultAuto);
     m_chooser.addOption("My Auto", kCustomAuto);
     SmartDashboard.putData("Auto choices", m_chooser);
-    
+    climb = new climb();
```

Comment:
The Climb class is capitalized.
```suggestion
    climb = new Climb();
```